tests/commonlib/kubernetes.py

```diff
@@ -223,7 +229,8 @@ def create_patched_resource(self, patch_resource_type, patch_body):
             patched_body = self.patch_resource_body(yml_resource, patch_body)
             created_resource = self.create_from_dict(patched_body, **relevant_metadata)
 
-            done = self.wait_for_resource(resource_type=resource_type, status_list=["RUNNING", "ADDED"], **relevant_metadata)
+            done = self.wait_for_resource(resource_type=resource_type, status_list=["RUNNING", "ADDED"],
+                                          **relevant_metadata)
```

Comment:
nit: usually when I split one arg by a new line, I split all of them
```suggestion
            done = self.wait_for_resource(resource_type=resource_type, 
                                          status_list=["RUNNING", "ADDED"],
                                          **relevant_metadata)
```

Reply:
done